src/components/CheckEditor/FormComponents/VariablesFields.tsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import { Controller, FieldPath, useFieldArray, useFormContext } from 'react-hook-form';
+import { GrafanaTheme2 } from '@grafana/data';
+import { Button, Field, IconButton, Input, Select, Stack, useStyles2 } from '@grafana/ui';
+import { css } from '@emotion/css';
+
+import { CheckFormValuesMultiHttp, MultiHttpVariableType } from 'types';
+import { useCheckFormContext } from 'components/CheckForm/CheckFormContext/CheckFormContext';
+import { MULTI_HTTP_VARIABLE_TYPE_OPTIONS } from 'components/constants';
+
+export const VariablesFields = ({ index }: { index: number }) => {
```

Comment:
Should we name this component something more specific to MultiHTTP? Sounds too generic otherwise.

Reply:
Done, good suggestion!